platforms/tktrex/extension/package.json

```diff
@@ -9,8 +9,8 @@
     "build:guardoni": "BUILD_TARGET=guardoni DEBUG=\"*\" webpack",
     "dist": " DEBUG=\"*:info,*:error\" NODE_ENV=production webpack",
     "dist:guardoni": "DEBUG=\"*:info,*:error\" NODE_ENV=production BUILD_TARGET=guardoni yarn build",
-    "watch": "DEBUG=\"*:info,*:error\" webpack --watch",
-    "watch:prod": "NODE_ENV=production webpack --watch",
+    "watch": "DOTENV_CONFIG_PATH=.env.development webpack --watch",
```

Comment:
This would also work also if you run `yarn tk:ext watch` from the root folder, or it is supposed to work only from the `extension/` folder?

Reply:
It works with `yarn tk:ext watch` (cause uses `__dirname`)